docs/docsite/rst/community/index.rst

```diff
@@ -34,10 +34,10 @@ Going deeper
 ============
 
 * I think Ansible is broken. How do I :ref:`report a bug <reporting_bugs>`?
-* I need functionality Ansible doesn't offer. How do I :ref:`request a feature <request_features>`?
+* I need functionality that Ansible doesn't offer. How do I :ref:`request a feature <request_features>`?
 * I'm waiting for a particular feature. How do I see what's :ref:`planned for future Ansible Releases <roadmaps>`?
 * I have a specific Ansible interest or expertise (for example, VMware, Linode, etc.). How do I get involved in a :ref:`working group <working_group_list>`?
-* I'd like to participate in conversations about features and fixes. How do I review GitHub issues and pull requests?
+* I'd like to participate in conversations about features and fixes. How do I review :ref:`GitHub issues and pull requests <community_documentation_contributions>`?
```

Comment:
Now the link goes to the top of the "Contributing to the Ansible Documentation" page!